applications/fishing-map/package.json

```diff
@@ -13,9 +13,9 @@
     "@globalfishingwatch/dataviews-client": "^3.0.0",
     "@globalfishingwatch/layer-composer": "^2.13.3",
     "@globalfishingwatch/linting": "^1.3.1",
-    "@globalfishingwatch/mapbox-gl": "1.12.0-gfw.1",
+    "@globalfishingwatch/mapbox-gl": "1.12.0-gfw.2",
     "@globalfishingwatch/react-hooks": "^3.0.0",
-    "@globalfishingwatch/react-map-gl": "^5.2.7-gfw.0",
+    "@globalfishingwatch/react-map-gl": "5.2.7-gfw.1-beta4",
```

Comment:
If we had issues with mapbox-gl version, won't we have the same with react-map-gl? because amathea is using [5.2.7-gfw.0](https://github.com/GlobalFishingWatch/frontend/pull/110/files#diff-0dcf200b9aba8bb33308f5cb7290344cR19)

Reply:
Honestly I don't really get it... With that setup Mapbox GL ends up with the correct versions in all instances in the `yarn.lock` file. 🤷‍♂️